client/src/test/java/org/asynchttpclient/reactivestreams/ReactiveStreamsTest.java

```diff
@@ -304,22 +304,27 @@ public void streamedResponseTest() throws Throwable {
 
     @Test(groups = "standalone")
     public void cancelStreamedResponseTest() throws Throwable {
-        try (AsyncHttpClient c = asyncHttpClient()) {
-
-            // Cancel immediately
-            c.preparePost(getTargetUrl()).setBody(LARGE_IMAGE_BYTES).execute(new CancellingStreamedAsyncProvider(0)).get();
+        // Cancel immediately
+        createClientAndCancelAfterN(0);
```

Comment:
This test is wrong. You shouldn't be closing and recreating a client instance in every step.
The purpose of this test is to check how a client instance recovers from those cancellations.

Reply:
Sorry, I think I misunderstood the intent of this test then and wanted to clean it up :)